sqlobject/pom.xml

```diff
@@ -123,6 +123,15 @@
                     </compilerArgs>
                 </configuration>
             </plugin>
+            <plugin>
+                <groupId>com.mycila</groupId>
+                <artifactId>license-maven-plugin</artifactId>
+                <configuration>
+                    <excludes>
+                        <exclude>src/test/resources/org/jdbi/v3/sqlobject/TestUseClasspathSqlLocator$Blanks/empty.sql</exclude>
```

Comment:
why are we adding this exception?  I'd prefer we just add the license to the files instead.

Reply:
Adding the license makes the file not empty :)

Reply:
Yeah, I don't think that is right.  a locator / template engine is *not* responsible for ensuring we get valid sql, only that we found all the files / strings / etc we look for.  An empty string is valid in that context, but will cause a problem later when you try to prepare it.

Reply:
Removing the emptiness check presents a backward compatibility problem though because the code is used in a fair few places